### DISCUSSION TITLE
Separate Note Skin For Opponent for KE 1.5.4?

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#2804] [Q&A] [answered]
I've always wondered how to give specific receptors different note skins. Or in my case right now, for the opponent in general. (receptors 0-3)
Does anyone know how to? I can't copy the source code of mods like [Vs Eteled](https://gamebanana.com/mods/313084) or [Bob & Bosip](https://gamebanana.com/mods/297087) because they use different versions. Plz, help me.

**Comment by ghost** [comment, marked as answer]
Nvm, I found out how.

**Reply by Adrian47roblox** [reply]
> Nvm, I found out how.

how?

**Reply by mamrofy** [reply]
how? lol

**Reply by RealDealJinx** [reply]
look here
https://gamebanana.com/tuts/14274
